packages/deployment/app/cumulus_api.template.yml

```diff
@@ -178,6 +178,9 @@ Resources:
       Environment:
         Variables:
           stackName: {{../stackName}}
+          system_bucket: {{../parent.system_bucket}}
```

Comment:
Do we need `system_bucket`?

Reply:
Nope, it's vestigial from the other method.